packages/core/src/lib/button/button.svelte

```diff
@@ -51,26 +48,26 @@ export let width: 'full' | 'default' = 'default';
   class={cx(
     'mx-auto inline-flex items-center justify-center gap-1.5 whitespace-nowrap border py-1.5 text-xs',
     {
-      'pointer-events-none select-none !border-disabled-light !bg-disabled-light text-disabled-dark':
+      '!border-disabled-light !bg-disabled-light text-disabled-dark pointer-events-none select-none':
```

Comment:
While we're cleaning up button: in the last component library I worked in, `pointer-events: none` on disabled buttons messed with tooltips, because `mouseenter` et. al. are pointer events. This was a problem because we wanted the tooltip to explain _why_ the button was disabled.

In general, I think `pointer-events: none` should be used really sparingly. In this specific case, since this is a native `<button>` and we are setting the disabled attribute, I don't think we need to disable pointer events to prevent the click handler from firing; it's built-in browser behavior.

(This comment probably applies to `select-none`, too, but I haven't used it before)

Reply:
I agree, and actually this current version makes our buttons pretty inaccessible. We probably shouldn't be using `disabled` since that will make buttons imperceivable to screen readers. `select-none` prevents users from selecting the button text, IMO we may just want that always on unless we feel it is useful to let users select/copy button text.